cache/remote_cache.go

```diff
@@ -8,13 +8,25 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/grafana/dskit/promregistry"
 )
 
 var (
 	_ Cache = (*MemcachedCache)(nil)
 	_ Cache = (*RedisCache)(nil)
 )
 
+const (
+	labelName             = "name"
+	labelBackend          = "backend"
+	backendRedis          = "redis"
+	backendMemcached      = "memcached"
+	cachePrefix           = "cache_"
```

Comment:
[nit] I would clarify which prefix it is.

```suggestion
	cacheMetricNamePrefix           = "cache_"
```